sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/models/CosmosItemIdentity.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.models;
+
+/**
+ * Encapsulates the PartitionKey/id tuple that uniquely identifies a CosmosItem
+ */
+public class CosmosItemIdentity {
```

Comment:
@FabianMeiswinkel shouldn't this class be marked with BETA annotation?

Reply:
We should discuss this. 
@FabianMeiswinkel  @moderakh - We have a release going out early next week, probably 9/8. 
If we can finalize this soon, we will be able to get this in.

Reply:
Fixed